apps/frontend/src/assets/css/variabels.css

```diff
@@ -0,0 +1,15 @@
+:root {
```

Comment:
I already created a file variables and added some colors there. Pull the changes from main and add your colors to the existing file